package.json

```diff
@@ -34,6 +34,11 @@
     ]
   },
   "dependencies": {
+    "@json-rpc-tools/utils": "^1.7.6",
+    "@walletconnect/client": "^2.0.0-beta.26",
+    "@walletconnect/utils": "^2.0.0-beta.26",
+    "alephium-walletconnect-provider": "0.1.2",
+    "alephium-web3": "0.9.0",
```

Comment:
Shouldn't we first decide what we do with the `@alephium/sdk` and `alephium-web3` before proceeding to merge this PR?

Reply:
Also, these depedencies should be added as devDepedencies (due to electron, only electron related depedencies should be added here)

Reply:
Interesting. I will move them to `devDepedencies`

Reply:
FWIW: Next time you can use `npm link` instead of deploying to npm to test packages